src/pydirectory/Directory.py

```diff
@@ -93,11 +98,28 @@ def createHiddenFolder(self):
                             " an autoDelete directory")
         return tempfile.mkdtemp(dir=self.path, prefix=".")
 
+    def listFilesWin(self):
+        output = []
+        for dirname, dirnames, filenames in os.walk(self.path):
+            for subdirname in dirnames:
+                output.append(os.path.join(dirname, subdirname))
+            for filename in filenames:
+                output.append(os.path.join(dirname, filename))
+        return output
+
+    def listFilesLinux(self):
+        find = "find %s -type f" % self.path
+        output = check_output(args=find.split()).strip().decode().split(
+            '\n')
+        return output
+
     def scan(self):
         self._files = {}
-        find = "find %s -type f" % self.path
-        output = check_output(args=find.split()).strip().decode().split('\n')
-        output = [line for line in output if not "__MACOSX" in line]
+        if system() is "Windows":
+            output = self.listFilesWin()
+        else:
```

Comment:
can you make this

output = self._listFilesWin() if system is "Windows" else self_listFilesLinux()

even better if you can define the following  up the top of the module
isWindows = system() is "Windows"
and use that